requirements.txt

```diff
@@ -20,7 +20,8 @@ sphinx
 rinohtype
 celery~=5.0.5
 redis==3.5.3
-psycopg2
+psycopg2-binary
 boto3==1.*
 sendgrid==6.6.*
-numpy==1.20.*
\ No newline at end of file
+numpy==1.20.*
+elasticsearch==7.12.*
```

Comment:
checkout elasticsearch dsl as per docs I shared initially and let me know your thoughts, 
its very native to the python implementation